pkg/controller/migmigration/restore.go

```diff
@@ -56,6 +56,46 @@ func (t *Task) ensureRestore() error {
 	return nil
 }
 
+// Ensure the first restore on the destination cluster has been
+// created  and has the proper settings.
+func (t *Task) ensureCopyRestore() error {
+	newRestore, err := t.buildRestore(nil)
```

Comment:
Again, wondering what the behavior of passing in `nil` rather than true/false to buildRestore() is. Feels weird.

Reply:
See above. `includePersistentVolumes` field on the backup spec expects a `*bool`